rsts/tutorials/run_flyte_hosted.rst

```diff
@@ -4,7 +4,7 @@
 [Optional] Run Your Workflow on a Hosted Flyte
```

Comment:
I thought you were going to move the contents of this file to a tab in the previous page?

Reply:
@kumare3 this PR is just warnings, but I can do it in the same one if you prefer?

Reply:
done